content/developer/reference/backend/testing.rst

```diff
@@ -819,35 +840,39 @@ driven explicitly by the user) it's more complicated when running
 "test" tours, or when running tours through the test suite. In that
 case there are two main tricks:
 
-- Have a step with a ``run() { debugger; }`` action.
+- Have a step property ``break: true,`` in debug mode (debug=True).
+
+  This adds a debugger at the start of the step and then allows you
+  to put breakpoints in the desired locations.
```

Comment:
I guessed you meant: 

This adds a debugger breakpoint at the start of the step. You can then add your own wherever you need.

Reply:
Yep